src/views/CharacterListView.js

```diff
@@ -1,20 +1,16 @@
 import React from "react";
 import { connect } from "react-redux";
-
+import { fetchAPI } from "../actions";
 import { CharacterList } from "../components";
 // import actions
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
-
   componentDidMount() {
-    // call our action
+    this.props.fetchAPI();
   }
 
   render() {
-    if (this.props.fetching) {
+    if (this.props.isLoading ? <h1> Loading CharacterList...</h1> : null) {
       // return something here to indicate that you are fetching data
     }
     return (
```

Comment:
```suggestion
else if (this.props.error) {
      return <div>{this.props.error}</div>
    }
    return (
```
I would add some error handling here, so your user knows what's going on when there is an error